appjs/views/edit_project.js

```diff
@@ -153,6 +154,45 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
     return ret;
   },
 
+  resizeForm: function(event){
+    var view = this;
+    if($(window).width() > 768){
+      logger.debug(event.target);
+      $(document).mousemove(function (event){
+        $('#embed-preview').addClass('screen');
+        if(event.pageX > 320 && $(window).width() - event.pageX > 300){
+          view.formWidth = $(window).width() - event.pageX;
+          $('#form-pane').css("width", view.formWidth);
+          $('#preview-pane').css("width", event.pageX);
+          view.showPreviewButtons();
+        }
+      });
+    }
+
+    $(document).mouseup(function (e) {
+      $('#embed-preview').removeClass('screen');
+      $(document).unbind('mousemove');
+    });
```

Comment:
Allows for user to pull/drag form in and out. Helpful when creating projects like the quiz where narrow forms can be a hassle.